src/main/java/com/twilio/http/ValidationClient.java

```diff
@@ -104,7 +107,14 @@ public Response makeRequest(Request request) {
 
         HttpMethod method = request.getMethod();
         if (method == HttpMethod.POST) {
-            builder.addHeader(HttpHeaders.CONTENT_TYPE, "application/x-www-form-urlencoded");
+            if(request.getContentType() != null && request.getContentType().equals("APPLICATION_JSON")){
+                HttpEntity entity = new StringEntity(request.encodeFormBody(), ContentType.APPLICATION_JSON);
```

Comment:
[self] : We can also include a json_payload body parameter in request to be sent here in entity, in place of `request.encodeFormBody()` . @sbansla thoughts ?